linchpin/provision/roles/libvirt/tasks/copy_image_src_remote.yml

```diff
@@ -7,7 +7,7 @@
   file:
     path: "{{ local_image_path }}"
     state: directory
-    mode: 0755
+    mode: 0770
```

Comment:
I understand why you took away access from the others, but why is the 770 necessary? If writing is necessary, the user rights would match first anyway. Thus, adding the second 7 there would only be for additional users of whatever group the remote_user belongs to and would provide full directory access to the 'local_image_path'.  Is that necessary?

Reply:
Like I stated in pull request, this is highly dependent on our setup and may not be appropriate in all environments. 

In our environment the remote_user (not root!) is a member of the qemu group and the premissions on the local_image_path looks like this:
drwxrwx---. 2 root qemu 4096 Jun  5 20:55 /var/lib/libvirt/images/linchpin/

I really don't have a general solution to this. Perhaps the requirement should be dropped altogether?

Reply:
@joanbe would just using the umask suffice here? We could just remove the mode line and see if that works? I think that would accomplish the same thing on both sides.

Reply:
Oh it works without the mode line

Reply:
Can you make that adjustment and let's see how the tests go. Thanks!

Reply:
Will do. Commit inbound.